libraries/AP_RangeFinder/AP_RangeFinder_LightWareSerial.cpp

```diff
@@ -74,8 +79,17 @@ bool AP_RangeFinder_LightWareSerial::get_reading(uint16_t &reading_cm)
         }
     }
 
-    // we need to write a byte to prompt another reading
-    uart->write('d');
+    uint32_t now = AP_HAL::millis();
+    if (now - last_init_ms > 1000 &&
+        now - state.last_reading_ms > 1000) {
+        // send enough serial transitions to trigger LW20 into serial
+        // mode
+        uart->write("www\r\n");
+        uart->begin(baudrate, 256, 256);
```

Comment:
Is there a reason why we "force" the buffer size here but not in init?

Reply:
i've confirmed it isn't needed (was added while debugging)